app/filters.js

```diff
@@ -10,6 +10,22 @@ marked.setOptions({
 module.exports = (env) => {
   const filters = {}
 
+  filters.push = (array, item) => {
+    array.push(item)
+    return array
+  }
+
+  /**
+   * Convert array to readable list format
+   * @param {Array} array Array to convert
+   * @example [A, B, C] => A, B and C
+   */
+  filters.formatOrList = (array = []) => {
+    const lf = new Intl.ListFormat('en', { style: 'short', type: 'disjunction' })
+    return lf.format(array)
+  }
```

Comment:
Does this do the same or similar to the `formatList` filter we have later in this file?

Reply:
It's similar, but uses "or" instead of "and" (set by `type: 'disjunction'`).

I'll update the comment though. 🤦

Reply:
Done in 90f94a4bde53d528ab8ac6b6e80b60eb70a04246

Reply:
Oooooh!